dev-lang/v8/v8-7.4.288.28.recipe

```diff
@@ -0,0 +1,132 @@
+SUMMARY="Google's open source JavaScript engine"
+DESCRIPTION="V8 is Google's open source JavaScript engine. V8 is written in \
+C++ and is used in Google Chrome, the open source browser from Google. V8 \
+implements ECMAScript as specified in ECMA-262, 5th edition, and runs on \
+Windows (XP or newer), Mac OS X (10.5 or newer), and Linux systems that use \
+IA-32, x64, or ARM processors. V8 can run standalone, or can be embedded into \
+any C++ application."
+HOMEPAGE="https://v8.dev/"
+COPYRIGHT="2006-2018 The V8 Project Authors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/v8/v8/archive/$portVersion.tar.gz"
+SOURCE_DIR="v8-$portVersion"
+CHECKSUM_SHA256="a9b1c74ff2fae728e1c94acf525321669679ef56da9cc4d781c427b26032bbf7"
+srcChromeVersion="74.0.3729.157"
+SOURCE_URI_2="https://gsdview.appspot.com/chromium-browser-official/chromium-$srcChromeVersion-lite.tar.xz"
+CHECKSUM_SHA256_2="d7406d4efba9916000f8a0f087e7eaf443523f264a59a51d2089c4b2e4c66be0"
+SOURCE_DIR_2="chromium-$srcChromeVersion"
+
+PATCHES="v8-$portVersion.patchset"
+PATCHES_2="build-$portVersion.patch"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	SECONDARY_ARCHITECTURES="x86"
+	ARCH=x86
+else
+	ARCH=x64
+fi
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+    commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	v8$secondaryArchSuffix = $portVersion
+	cmd:d8$commandSuffix = $portVersion
+	lib:libv8$secondaryArchSuffix = $portVersion
+	lib:libv8_for_testing$secondaryArchSuffix = $portVersion
+	lib:libv8_libbase$secondaryArchSuffix = $portVersion
+	lib:libv8_libplatform$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexecinfo$secondaryArchSuffix
+	lib:libssp$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	v8${secondaryArchSuffix}_devel = $portVersion
+	devel:libv8$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	v8$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	gcc${secondaryArchSuffix}_syslibs
+	gcc${secondaryArchSuffix}_syslibs_devel
+	llvm7$secondaryArchSuffix
+	llvm7${secondaryArchSuffix}_libs
+	devel:libgcc$secondaryArchSuffix
+	devel:libexecinfo$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:clang
+	cmd:clang++
```

Comment:
It appears V8 can be built with GCC >= 6.3.0. Please remove all the Clang stuff and let's use that by default.

Reply:
V8 recipe now uses GCC instead of Clang.